malware/network/cnp-ingress-deny-syslogk.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: "cilium.io/v2"
+kind: ClusterWideNetworkPolicy"

```

Comment:
@tamilmaran-7 
This should be `CiliumClusterwideNetworkPolicy`
and the format is also wrong. please refer https://docs.cilium.io/en/stable/policy/language/#host-policies